scripts/pre_push_hook.py

```diff
@@ -226,6 +227,18 @@ def extract_files_to_lint(file_diffs):
     return lint_files
 
 
+def get_remote_branch_name():
+    """Returns remote branch name against which the diff has to be checked.
+
+    Returns:
+        str: The name of the remote branch.
+    """
+    if common.is_current_branch_a_hotfix_branch():
```

Comment:
Shouldn't this be the case for both release branches and hotfix branches?

Reply:
This will create an issue when we push the release branch the first time - since the branch won't exist on upstream.